src/main/java/org/sefglobal/scholarx/config/SecurityConfig.java

```diff
@@ -0,0 +1,21 @@
+package org.sefglobal.scholarx.config;
+
+import org.springframework.security.config.annotation.web.builders.HttpSecurity;
+import org.springframework.security.config.annotation.web.configuration.EnableWebSecurity;
+import org.springframework.security.config.annotation.web.configuration.WebSecurityConfigurerAdapter;
+
+@EnableWebSecurity
+public class SecurityConfig extends WebSecurityConfigurerAdapter {
+
+    @Override
+    protected void configure(HttpSecurity http) throws Exception {
+        http
+                .csrf().disable()
+                .authorizeRequests()
+                .antMatchers("/admin/**").authenticated()
```

Comment:
Permit the admin requests for now since this will fail the test cases with `403`. 
After the user authentication is added we can authenticate the path and the test cases.

Reply:
Right. I have updated it to permitAll